data/shaders/basic_vs.glsl

```diff
@@ -1,13 +1,22 @@
 #version 150 core
-#include <locals>
 #include <globals>
 
 in vec4 a_Position;
 in vec4 a_Normal;
 in vec2 a_TexCoord;
 out vec2 v_TexCoord;
+out vec4 v_Color;
+
+in vec4 i_World0;
+in vec4 i_World1;
+in vec4 i_World2;
+in vec4 i_World3;
```

Comment:
as I mentioned before (and I don't think there was any reply?), we should use 3 vectors instead of 4 for the world transformation.

Reply:
Will do